pvlib/irradiance.py

```diff
@@ -1043,7 +1047,23 @@ def perez(surface_tilt, surface_azimuth, dhi, dni, dni_extra,
     else:
         sky_diffuse = np.where(np.isnan(airmass), 0, sky_diffuse)
 
-    return sky_diffuse
+    if return_components:
+        component_keys = ('isotropic', 'circumsolar', 'horizon')
+        diffuse_components = dict.fromkeys(component_keys)
```

Comment:
Sure (see following commit). I was just wondering why we wanted to use an ordered dictionary instead of a regular one here?